py-polars/requirements-dev.txt

```
@@ -47,6 +47,8 @@ zstandard
 # Plotting
 hvplot>=0.9.1
 matplotlib
+# Styling
+great-tables>=0.8.0
```

does it work to do
```suggestion
great-tables>=0.8.0; python_version >= '3.9'
```
?

it's only 4 months til Python3.8 EOL so maybe that's fine?